fixed-requirements.txt

```diff
@@ -74,6 +73,5 @@ nose-parallel==0.4.0
 psutil==5.8.0
 python-dateutil==2.8.1
 python-statsd==2.1.0
-ujson==1.35
```

Comment:
ujson is only used for benchmark tests now. We use orjson everywhere else.
https://github.com/StackStorm/st2/blob/6cd582e65f3ef8d7b1ad7c7b3df47e632de9f67b/test-requirements.txt#L45